vignettes/simple-forecasts.Rmd

```diff
@@ -17,6 +17,7 @@ knitr::opts_chunk$set(
 ```
 
 ```{r pkgs}
+devtools::load_all('.')
```

Comment:
Can you delete this line? It's not needed when the package is installed.

Reply:
resolved!